lib/Myriad.pm

```diff
@@ -628,6 +629,8 @@ async method run () {
         }))
     }
 
+    $self->storage;
```

Comment:
This looks wrong - what's it doing? If it's setting up the initial storage, would expect `await` or at least a comment.

Reply:
yes indeed, this a master hack fixing this case is what I'm working on next, but for historical accuracy, things are going to be handled in the `on_startup` processing so no need to `await` this is basically to just instantiate the object